website/www/site/layouts/index.html

```diff
@@ -85,49 +70,108 @@
   </div>
 {{ end }}
 
-{{ define "logos-section" }}
-  <div class="logos section">
-    <div class="logos__title">
-      {{ T "home-logos-title" }}
-    </div>
-    <div class="logos__logos">
-      {{ range $logo := $.Site.Data.works_with }}
-      <div class="logos__logos__logo">
-        <a href="{{ $logo.url }}"><img src="{{ $logo.image_url }}" alt="{{ $logo.title }}"></a>
+{{/*
+  TODO:
+    This is the implementation of the design.
+    The event sync functionality will be implemented when we have the event page.
```

Comment:
We need to implement event page and event sync. @Jakub-Sadowski is there a PR for this?

Reply:
implemented in https://github.com/apache/beam/pull/13596